tests/storage/fsp/conftest.py

```diff
@@ -0,0 +1,23 @@
+import logging
+import pytest
+
+@pytest.fixture(scope='session')
+def fsp_config(host):
+    pathtodb = host.ssh(['mktemp', '-d'])
+    pathtoshareddir = host.ssh(['mktemp', '-d'])
+    return {
+        'pathtodb': pathtodb,
+        'pathtoshareddir': pathtoshareddir
+    }
```

Comment:
`pathtodb` => `path_to_db` or even better: `db_path`
`pathtosharedir` => `path_to_sharedir` or `shared_dir_path`

More generally, `snake_case` is the convetion for naming in python. :)